src/backend/core/authentication/backends.py

```diff
@@ -90,7 +106,29 @@ def get_or_create_user(self, access_token, id_token, payload):
                 raise SuspiciousOperation(_("User account is disabled"))
             self.update_user_if_needed(user, claims)
         elif self.get_settings("OIDC_CREATE_USER", True):
-            user = User.objects.create(sub=sub, password="!", **claims)  # noqa: S106
+            user = self.create_user(claims)
+
+        # Data cleaning, to be removed when user organization is null=False
+        # or all users have an organization.
+        if not user.organization_id:
+            domain = get_domain_from_email(email)
+            try:
+                organization, organization_created = Organization.objects.get_or_create(
+                    siret=claims.get("siret"),
+                    domain=domain,
+                )
+                if organization_created:
+                    logger.info("Organization %s created", organization)
+                    # For this case, we don't create an OrganizationAccess we will
+                    # manage this later.
```

Comment:
It's not handled on purpose, because I don't the first user who login after the release to be the admin of their organization. So we will keep organization without admin and we will have to manually clean things up (while there is not that much organization in the database). I will improve the comment.